### PR TITLE
Better error handling

### PR DESCRIPTION
**Pull request by odeimaiz**
## What do these changes do?
- Error page:
  - Show human friendly error code text
  - More space for displaying entire error message

- Unresponsive backend:
  - Improve error handling
  - Add "Please, try again later" to 404 and 503 error messages

image
webserverdown

## Related issue/s
closes https://github.com/ITISFoundation/osparc-simcore/issues/2283
closes https://github.com/ITISFoundation/osparc-simcore/issues/2350

### CODE REVIEW

File: services/web/client/source/class/osparc/Error.js [hunk 1]
```diff
@@ -51,6 +51,52 @@ qx.Class.define("osparc.Error", {
...
+      "409": "Conflict",
+      "410": "Gone",
+      "411": "Length Required",
+      "412": "Precondition Required",
+      "413": "Request Entry Too Large",
+      "414": "Request-URI Too Long",
+      "415": "Unsupported Media Type",
+      "416": "Requested Range Not Satisfiable",
+      "417": "Expectation Failed",
+      "418": "I'm a teapot",
```
Comment (pcrespov) [inline services/web/client/source/class/osparc/Error.js:89]: 418 is my favorie

Comment (pcrespov) [reply services/web/client/source/class/osparc/Error.js:89]: TIP:  ``https://httpstatuses.com/{status_code}``  give a nice explanation of those. Might add a link to the message ?

**Review by pcrespov — APPROVED** [review]
:tada:

File: services/web/client/source/class/osparc/Error.js (hunk 1 above)
Comment (odeimaiz) [reply services/web/client/source/class/osparc/Error.js:89]: Too techie imo

**Review by GitHK — APPROVED** [review]
I see what you did here. It exposes a bit more of the underlying structure, but it works. Thanks a lot for this 👍

**Review by sanderegg — APPROVED** [review]
great addition!

**Pull request merged**